superset/dashboards/dao.py

```diff
@@ -35,6 +36,14 @@ class DashboardDAO(BaseDAO):
     model_cls = Dashboard
     base_filter = DashboardFilter
 
+    @staticmethod
+    def get_charts_for_dashboard(dashboard_id: int) -> List[Slice]:
```

Comment:
This method does not return `List[Slice]` but `List[Dict[str, Any]]`

Reply:
Oh yes, thanks